.gitignore

```diff
@@ -1,4 +1,5 @@
+.idea/
```

Comment:
Local folder?

Reply:
Jup. JetBrains default project folder for all their IDE's. Can recommend their suite.

I'd like to keep that, if you don't mind. Had to many git's being scrambled accidentally.